src/routes/cartpage/cartpage.css

```diff
@@ -0,0 +1,43 @@
+.cartpage-container{
+    display: grid;
+    grid-template-columns: 6fr 4fr;
+    grid-template-rows: auto 1fr;
+}
+.cartpage-count{
+    grid-column: 1/3;
+    grid-row: 1/2;
+    padding: 1rem;
+    text-align: center;
+}
+.cartpage-aside{
+    grid-column: 2/3;
+    grid-row: 2/3;
+}
+.cartpage-main{
+    grid-column: 1/2;
+    grid-row: 2/3;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+@media (max-width:500px) {
+    .cartpage-container{
+        display: flex;
+        flex-direction: column;
+    }
+    .cartpage-count{
+
+    }
```

Comment:
you can remove this if forgotten to remove.

Reply:
sure